Gemfile.lock

```diff
@@ -89,6 +89,12 @@ PATH
     samson_jenkins (0.0.0)
       jenkins_api_client (~> 1.3)
 
+PATH
+  remote: plugins/jira
+  specs:
+    samson_jira (0.0.0)
```

Comment:
should use released version, right?

Reply:
always stays at 0.0.0 since we don't version plugins (yet)